Sources/Profile-View.php

```diff
@@ -833,10 +833,14 @@ function showPosts($memID)
 	// Shortcut used to determine which $txt['show*'] string to use for the title, based on the SA
 	$title = array(
 		'attach' => 'Attachments',
-		'unwatchedtopics' => 'Unwatched',
 		'topics' => 'Topics'
```

Comment:
sorry this comment is unrelated to the pr but, is it me or are these hardcoded strings ?

Reply:
Yes they are

Reply:
'Attachments' can be replaced for $txt['showAttachments'], 'Topics' can be replaced for $txt['showTopics'] and 'Unwatched' for $txt['showUnwatched'].

Reply:
Oh I see it now, indeed.